discord_slash/client.py

```diff
@@ -300,14 +304,17 @@ async def sync_all_commands(self, delete_from_unused_guilds=False):
         If ``sync_commands`` is ``True``, then this will be automatically called.
 
         :param delete_from_unused_guilds: If the bot should make a request to set no commands for guilds that haven't got any commands registered in :class:``SlashCommand``
+        :param delete_perms_from_unused_guilds: If the bot should make a request to clear permissions for guilds that haven't got any permissions registered in :class:``SlashCommand``
         """
+        permissions_map = {}
         cmds = await self.to_dict()
         self.logger.info("Syncing commands...")
         cmds_formatted = {None: cmds['global']}
         for guild in cmds['guild']:
             cmds_formatted[guild] = cmds['guild'][guild]
-                
+
```

Comment:
Is this an intentional formatting change that is required?

Reply:
I think my ide removes trailing spaces, and also I believe this aligns with the check style you all use.